firefly1:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  firefly1/arm1_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly1/arm1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly1/arm2_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly1/arm2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly1/arm3_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly1/arm3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}



firefly2:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  firefly2/arm1_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly2/arm1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly2/arm2_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly2/arm2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly2/arm3_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly2/arm3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}



firefly3:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  firefly3/arm1_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly3/arm1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly3/arm2_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly3/arm2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly3/arm3_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly3/arm3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}



firefly4:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  firefly4/arm1_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly4/arm1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly4/arm2_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly4/arm2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}

  firefly4/arm3_joint_position_controller:
    type: position_controllers/JointPositionController
    joint: firefly4/arm3_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
